src/components/__tests__/NavBar.test.tsx

```diff
@@ -13,7 +13,7 @@ it("renders the navbar correctly", () => {
     .create(
       <Provider store={store}>
         <StaticRouter>
-          <NavBar location={{ pathname: "/" }} />
+          <NavBar collapsed={true} toggleCollapse={() => {}} />
```

Comment:
I might be wrong about this one, too, but I think it can just be:

```suggestion
          <NavBar collapsed=true toggleCollapse={() => {}} />
```